lib/cocoapods/installer/project_cache/project_cache_analyzer.rb

```diff
@@ -150,6 +154,13 @@ def compute_dirty_targets(targets)
             support_files_dir_exists && xcodeproj_exists
           end
         end
+
+        def compute_sibling_pod_targets(all_pod_targets, targets)
+          pod_names = targets.map(&:pod_name)
+          all_pod_targets.select do |target|
```

Comment:
It might be cleaner to do something like:
```ruby
pod_targets_by_name = pod_targets.group_by { |target| target.pod_name }
sibling_targets_to_generate = pod_targets_to_generate.flat_map { |t| pod_targets_by_name[t.pod_name] }
```
And then keep the `.uniq` above where we assign `pod_targets_to_generate =`